internal/olm/client.go

```diff
@@ -72,8 +72,13 @@ func (c Client) InstallVersion(ctx context.Context, version string) (*olmresourc
 	objs := toObjects(resources...)
 
 	status := c.GetObjectsStatus(ctx, objs...)
-	if status.HasExistingResources() {
-		return nil, errors.New("detected existing OLM resources: OLM must be completely uninstalled before installation")
+	installed, err := status.HasInstalledResources()
+	if installed {
+		return nil, errors.New("detected existing OLM resources: OLM must be completely uninstalled before installation" +
+			fmt.Sprintf("\nResources:\n%s", status.String()))
```

Comment:
Nit: use `fmt.Errorf()` here instead of `errors.New()` and `fmt.Sprintf()`? 

Also, we should be able to use just `status` instead of `status.String()` since `%s` will use a struct's `String()` method. 

Lastly, to match output of `operator-sdk alpha olm status`, maybe there's no need for `Resources:` in the output? Just an empty line instead? WDYT?

Ditto below.

Reply:
I think we can remove the `Resources: ...` bit since `olm status` will print the status of an existing OLM deployment correctly in this PR.